### ISSUE TITLE
Missing section on configuring BP / full nodes

### ISSUE THREAD

**Issue by aspiers** [aergoio/guide#14]
https://docs.aergo.io/en/latest/running-node/configuration.html says:

> There are three common modes of operation for aergo server:
>
> - BP Node - Block provider node, which mines and propagates blocks
> - Full Node - Node to get and validate all blocks
> - Single Node - BP Node without peer, which uses to something like test.
>
> We describe only single node configuration in this. More will updated for the BP Node and full node.

So currently it is not clear how to join e.g. the testnet.  I see that #4 was closed but not sure what the resolution was.

**Comment by graup** [comment]
You're right, that page is missing some minimal config file for how to sync with the testnet. Normally you don't need it (the [default config](https://docs.aergo.io/en/latest/using-network/syncing.html) should suffice), but we should still explain how to do it if you want to.
